crates/smtp/src/inbound/session.rs

```diff
@@ -136,7 +136,10 @@ impl<T: AsyncWrite + AsyncRead + IsTls + Unpin> Session<T> {
                                 self.handle_expn(value).await?;
                             }
                             Request::StartTls => {
-                                if !self.stream.is_tls() {
+                                if self.instance.tls_acceptor.is_none() {
+                                    self.write(b"502 5.5.1 Command not implemented.\r\n")
```

Comment:
Can we explicitly return what command is not implemented?

Reply:
Not sure, I copied the string returned by `ETRN`/`ATRN`/`BURL` and it's also the exact response that Postfix gives.

Reply:
Okay, let's not change it then. Thank you for commenting quickly